File: src/test/java/ru/capjack/degos/publish/DegosPublishPluginTest.java
package ru.capjack.degos.publish;

import org.gradle.api.Project;
import org.gradle.testfixtures.ProjectBuilder;
import org.junit.jupiter.api.Test;

class DegosPublishPluginTest {
	@Test
	void simpleApply() {
		Project project = ProjectBuilder.builder().build();
		project.getPluginManager().apply(DegosPublishPlugin.class);
	}
}

File: src/test/kotlin/ru/capjack/degos/publish/DegosPublishPluginTest.kt
package ru.capjack.degos.publish

import org.gradle.testfixtures.ProjectBuilder
import org.junit.jupiter.api.Test

internal class DegosPublishPluginTest {
	@Test
	fun `Simple apply`() {
		val project = ProjectBuilder.builder().build()
		project.pluginManager.apply(DegosPublishPlugin::class.java)
	}
}